samples/MvcSample.Web/Controllers/JsonPatchController.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.AspNet.JsonPatch;
+using Microsoft.AspNet.Mvc;
+
+// For more information on enabling Web API for empty projects, visit http://go.microsoft.com/fwlink/?LinkID=397860
```

Comment:
This got added when I created `webapi controller` using template. Should I remove it?